tests/parser_test.toit

```diff
@@ -333,15 +333,23 @@ test-invert-non-flag:
     cmd.run ["--no-foo"] --ui=ui
 
 test-value-for-flag:
+  expected/Map? := null
   cmd := cli.Command "test"
       --options=[
         cli.Flag "foo" --short-name="f",
       ]
       --run=:: | app/cli.Application parsed/cli.Parsed |
-        unreachable
+        check-arguments expected parsed
+
+  expected = {"foo": true}
+  cmd.run ["--foo=true"]
+
+  expected = {"foo": false}
+  cmd.run ["--foo=false"]
 
-  expect-abort "Cannot specify value for boolean flag --foo.": | ui/cli.Ui |
-    cmd.run ["--foo=bar"] --ui=ui
+  ["on", "off", "yes", "no"].do: | value |
```

Comment:
Test with " true" and "false "?

Reply:
I misunderstood. Testing true and false with leading/trailing spaces here:
https://github.com/toitlang/pkg-cli/pull/68